xml/Microsoft.VisualC.StlClr/ITree`2.xml

```diff
@@ -527,11 +539,18 @@
       </Parameters>
       <Docs>
         <param name="">To be added.</param>
```

Comment:
The parameter name and description are missing. The parameter name is A_1. That's also true of one of the other overloads of insert, where the parameter name is also A_1.